packages/fhir-converter/src/templates/cda/Resources/Observation.hbs

```diff
@@ -70,29 +70,20 @@
         {{#if observationEntry.value.code}}
             "valueCodeableConcept":{{>DataType/CodeableConcept.hbs code=observationEntry.value}},
         {{else if observationEntry.value.value}}
-            "valueQuantity":
-            {
-                "value":"{{observationEntry.value.value}}",
-                "unit":"{{observationEntry.value.unit}}",
-            }, 
+            {{#with (extractRangeFromQuantity observationEntry.value) as |rangeObj|}}
```

Comment:
if range is valid, then we take both sides of the "-" and pass them through the whole Quantity/SimpleQuantity process via the `Range.hbs` data type